app/tests/distributions/test_attribution_model.py

```diff
@@ -0,0 +1,130 @@
+    assert actual.name_fr == attribution["name_fr"]
+    assert actual.name_en == attribution["name_en"]
+    assert actual.name_it == attribution["name_it"]
+    assert actual.name_rm == attribution["name_rm"]
+
+    assert actual.description_de == attribution["description_de"]
+    assert actual.description_fr == attribution["description_fr"]
+    assert actual.description_en == attribution["description_en"]
+    assert actual.description_it == attribution["description_it"]
+    assert actual.description_rm == attribution["description_rm"]
+
+    assert actual.provider.acronym_de == "ENSI"
+
+
+def test_object_created_in_db_with_optional_fields_null(db):
+    attribution = {
+        "name_de": "BAFU",
+        "name_fr": "OFEV",
+        "name_en": "FOEN",
+        "name_it": None,
+        "name_rm": None,
+        "description_de": "Bundesamt für Umwelt",
+        "description_fr": "Office fédéral de l'environnement",
+        "description_en": "Federal Office for the Environment",
+        "description_it": None,
+        "description_rm": None,
+        "provider": Provider.objects.create(acronym_de="ENSI")
+    }
+    Attribution.objects.create(**attribution)
+
+    attributions = Attribution.objects.all()
+
+    assert len(attributions) == 1
+
+    actual = Attribution.objects.last()
+    assert actual.name_de == attribution["name_de"]
+    assert actual.name_fr == attribution["name_fr"]
+    assert actual.name_en == attribution["name_en"]
+    assert actual.name_it == attribution["name_it"]
+    assert actual.name_rm == attribution["name_rm"]
+
+    assert actual.description_de == attribution["description_de"]
+    assert actual.description_fr == attribution["description_fr"]
+    assert actual.description_en == attribution["description_en"]
+    assert actual.description_it == attribution["description_it"]
+    assert actual.description_rm == attribution["description_rm"]
+
+
+def test_raises_exception_when_creating_db_object_with_mandatory_field_null(db):
+    provider = Provider.objects.create()
+    with raises(IntegrityError):
+        Attribution.objects.create(name_de=None, provider=provider)
+
+
+def test_form_valid_for_blank_optional_field(db):
+
+    class AttributionForm(ModelForm):
+
+        class Meta:
+            model = Attribution
+            fields = "__all__"
+
+    provider = Provider.objects.create()
+
+    data = {
+        "name_de": "BAFU",
+        "name_fr": "OFEV",
+        "name_en": "FOEN",
+        "description_de": "Bundesamt für Umwelt",
+        "description_fr": "Office fédéral de l'environnement",
+        "description_en": "Federal Office for the Environment",
+        "provider": provider.id,
+    }
+    form = AttributionForm(data)
+
+    assert form.is_valid() is True
+
+
+def test_form_invalid_for_blank_mandatory_field(db):
+
+    class AttributionForm(ModelForm):
+
+        class Meta:
+            model = Attribution
+            fields = "__all__"
+
+    provider = Provider.objects.create()
```

Comment:
I guess you could theoretically omit the `db` fixture here and create the Model via `Provider` 😅

Reply:
moved to fixture